src/databricks/labs/ucx/install.py

```diff
@@ -971,6 +986,15 @@ def _get_cluster_policies_with_external_hive_metastores(self):
                     yield policy
                     break
 
+    def _check_policy_has_instance_pool(self, policy_id):
```

Comment:
Too much code in this module, please extend with another module.

Reply:
Same here, the logic is pretty simple, not sure how to break it